src/go/k8s/apis/core/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package v1alpha1 contains API Schema definitions for the core v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=core.vectorized.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion	= schema.GroupVersion{Group: "core.vectorized.io", Version: "v1alpha1"}
```

Comment:
what's `core`

Reply:
maybe let's go with redpanda.vectorized.io ?

Reply:
I was thinking about the group that is tightly coupled with cluster setup. Other groups could be:

- `rbac` or `tenant` where for 1 cluster there could be multiple users with multiple roles to access admin API, kafka API and other lower level things 
- `wasm` definition of the additional stream processors
- `ingress` definition of how cluster would be accessible outside of an kubernetes cluster
- `topic` definition which could declaratively create topics and partitions. The operators would block access to the admin API, because of that `topic` group.

I'm open to rename the `core` group or remove it and use flat structure. WDYT?

Reply:
I think we can always add that on top of the redpanda group if we need to, so it would be 'topic.redpanda.vectorized.io'. At least my preference would be to go that way...

Reply:
right. 'core' doesn't mean anything to me. redpanda does.

Reply:
Ok I will address this in the next `git push`